Crypto landing page copy/css/style.css

```diff
@@ -0,0 +1,995 @@
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.left-block {
+    display: flex;
+    align-items: center;
+}
+
+.header-title {
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 600;
+    font-size: 23px;
+    line-height: 28px;
+    text-transform: capitalize;
+    color: #FFFFFF;
+}
+
+.right-block {
+    display: flex;
+    align-items: center;
+    gap: 15px;
+}
+
+.right-block ul {
+    display: flex;
+    justify-content: space-around;
+    gap: 30px;
+    list-style-type: none;
+    text-decoration: none;
+}
+
+.right-block a {
+    text-decoration: none;
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 30px;
+    text-transform: capitalize;
+    color: #767799;
+}
+
+.header-button a {
+    display: block;
+    border: 2px solid #FFFFFF;
+    padding: 10px 40px 10px 40px;
+    color: #FFFFFF;
+}
+
+/* Section 1 */
+
+.section1 {
+    display: block;
+}
+
+.container-fluid {
+    display: flex;
+    justify-content: space-between;
+    gap: 10px;
+}
+
+.elips {
+    display: block;
+    position: relative;
+    max-width: 554px;
+    margin-top: 183px;
+
+}
+
+.section1-img {
+    max-width: 572px;
+    margin-top: 73px;
+    height: auto;
+}
+
+.elips-img img {
+    top: 11%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 800px;
+    height: 800px;
+    display: block;
+    position: absolute;
+    z-index: -1;
+    object-fit: cover;
+}
+
+.elips h1 {
+    font-family: var(--font-primary);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 90px;
+    display: flex;
+    align-items: center;
+    text-transform: lowercase;
+    color: #FFFFFF;
+    position: relative;
+}
+
+.elips p {
+    margin-top: 25px;
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 168.5%;
+    color: #767799;
+}
+
+.elips h1::before {
+    content: '';
+    background: url('../img/strelka.png') no-repeat right top, url("../img/strelka2.png") no-repeat left bottom;
+    background-size: contain, auto;
+    width: 100px;
+    height: 53px;
+    transform: rotate(3deg);
+    position: absolute;
+    top: -42px;
+    right: 73px;
+}
+
+.section1-bottom {
+    display: flex;
+    flex-direction: row;
+    gap: 87px;
+    margin-top: 58px;
+}
+
+.section1-bottom img {
+    display: block;
+}
+
+
+.section1-bottom a {
+    display: inline-block;
+    border: 1px solid #FFFFFF;
+    text-decoration: none;
+    padding: 20px 44.5px 20px 44.5px;
+    font-family: var(--font-secondary);
+    font-style: normal;
+    font-weight: 600;
+    font-size: 21px;
+    line-height: 25px;
+    text-transform: capitalize;
+    color: #FFFFFF;
+}
+
+/* Sectiion 2 */
+
+.section2 {
+    background: #162446;
+}
+
+.section2-wrapper {
+    display: block;
+    padding-top: 41px;
+    padding-bottom: 41px;
+
+}
+
+.section2-wrapper ul {
```

Comment:
you have images inside ul, if you upload image like 2000px width, it will have such size. 
add styles like:
```
.section2-wrapper .item { display: block; max-width: 200px; }
.section2-wrapper .item img { display: block; width: 100%; height: auto;}
```